inference-engine/src/gna_plugin/layers/gna_layer_type.cpp

```diff
@@ -60,6 +60,22 @@ bool GNAPluginNS::AreLayersSupported(InferenceEngine::CNNNetwork& network, std::
                                                                 ", and batch size(" + std::to_string(batch_size) + ") != 1 not supported";
                                                    check_result =  false;
                                                }
+                                               if (LayerInfo(layer).isFullyConnected()) {
```

Comment:
I don't think that it is the right place for these checks. Can we move to the gna_limitation file or find new place where all limitations will be checked? There is a big mess with it.

Reply:
done